nemo/utils/decorators/experimental.py

```diff
@@ -24,13 +24,12 @@ def experimental(cls):
     """
 
     def wrapped(cls):
-        class WrappedClass(cls):
-            def __init__(self, *args, **kwargs):
-                super().__init__(*args, **kwargs)
-                logging.warning(
-                    f'Module {cls} is experimental, not ready for production and is not supported. Use at your own risk.'
-                )
-
-        return WrappedClass
+        logging.warning(
+            f'Module {cls} is experimental, not ready for production and is not fully supported. Use at your own risk.'
+        )
+
+        cls.__name__ = cls.__name__ + '(Experimental)'
```

Comment:
as @tkornuta-nvidia suggested on slack, let's just keep warning  and not change class name

Reply:
Fixed below